watchtower/wtserver/server.go

```diff
@@ -380,14 +389,14 @@ func (s *Server) handleCreateSession(peer Peer, id *wtdb.SessionID,
 	// Assemble the session info using the agreed upon parameters, reward
 	// address, and session id.
 	info := wtdb.SessionInfo{
-		ID:            *id,
-		RewardAddress: rewardAddrBytes,
+		ID: *id,
 		Policy: wtpolicy.Policy{
 			BlobType:     req.BlobType,
 			MaxUpdates:   req.MaxUpdates,
 			RewardRate:   req.RewardRate,
 			SweepFeeRate: req.SweepFeeRate,
 		},
+		RewardAddress: rewardScript,
```

Comment:
To make things more consistent we should likely rename these fields to be `RewardPkScript`, just to be more consistent and eliminate any ambiguity. Not a blocker, though can be done in a follow up PR as the rename would likely sprawl multiple files.